packages/scoutgame-ui/src/components/scout/ScoutPage.tsx

```diff
@@ -26,17 +33,14 @@ export const scoutTabMobileOptions: TabItem[] = [
   { label: 'New Scouts', value: 'new-scouts' }
 ];
 
-export function ScoutPage({
+export async function ScoutPage({
```

Comment:
what is the purpose of making this async/server-only?

Reply:
We have child components that are async/server-only